lib/Controller/LoginDeniedController.php

```diff
@@ -27,13 +27,15 @@
 use OCP\AppFramework\Http;
 use OCP\AppFramework\Http\TemplateResponse;
 
+/**
+ * @api
```

Comment:
What is this about?

Reply:
`psalm` complaining the class isn't used. 
There is also `@psalm-api` which is maybe more explicit :)